jobs/build/refresh-images/Jenkinsfile

```diff
@@ -91,6 +91,35 @@ node('openshift-build-1') {
                         error("Version overrides must start with 'v'")
                     }
                     oit_update_docker_args = "--version ${VERSION_OVERRIDE}"
+
+                    // Get the OCP version from the current build
+                    // query-rpm-version returns "version: v3.10.0" for example
+                    def detected_version = buildlib.oit("""
+--working-dir ${OIT_WORKING} --group 'openshift-${OSE_MAJOR}.${OSE_MINOR}'
+--quiet
+images:query-rpm-version
+--repo-type unsigned
+""", [capture: true]).split(' ')[1]
+
+                    def proceed = input(
+                        message: """\
+Remember to rebuild signed puddles before proceeding. 
+You have specified version: ${VERSION_OVERRIDE}
+oit has detected the signed puddle contains: ${detected_version}
+Proceed?
+""",
+                        parameters: [
```

Comment:
A naked input statement will prompt the user to proceed or abort
e.g. `input "Are you certain you want to =====>UPGRADE<===== the =====>${CLUSTER_NAME}<===== cluster?"`

will render as (where `Proceed` and `Abort` are hyperlinks):
```
Are you certain you want to =====>UPGRADE<===== the =====>free-int<===== cluster?
Proceed or Abort
```
So you should be able to remove the parameters and `if (!proceed) { ... }` block